src/js/containers/movie-tile/MovieTile.jsx

```diff
@@ -14,7 +14,7 @@ class MovieTile extends Component {
         <div>
```

Comment:
From this file: 

`const movieDetails = this.props.movieDetails;` 
you can make it more easy and neat with object destructing 
https://dev.to/sarah_chima/object-destructuring-in-es6-3fm
`const {movieDetails} = this.props`